com.unity.shadergraph/Editor/GraphDeltaRegistry/Generation/Interpreter.cs

```diff
@@ -75,11 +75,12 @@ internal static Block EvaluateGraphAndPopulateDescriptors(NodeHandler rootNode,
                 {
                     if (port.IsHorizontal && port.IsInput)
                     {
-                        var entry = port.GetTypeField().GetSubField<IContextDescriptor.ContextEntry>(ShaderGraph.Registry.Types.GraphType.kEntry).GetData();
+                        var name = port.ID.LocalPath;
+                        var type = registry.GetTypeBuilder(port.GetTypeField().GetRegistryKey()).GetShaderType(port.GetTypeField(), container, registry);
                         var varOutBuilder = new BlockVariable.Builder(container)
                         {
-                            Name = entry.fieldName,
-                            Type = EvaluateShaderType(entry, container)
```

Comment:
Is `EvaluateShaderType` still relevant and something that will be used in the future? Cause currently I see that there are no usages of it, might be worth removing if its dead code

Reply:
EvaluateShaderType is an artifact of early context code, so yea-- it'll get removed at some point.